lib/tab-bar-item-ios.js

```diff
@@ -16,18 +16,22 @@ export default function createTabBarItemIOSComponent(IconNamePropType, getImageS
       iconName: IconNamePropType.isRequired,
       selectedIconName: IconNamePropType,
       iconSize: PropTypes.number,
+      iconColor: PropTypes.string,
+      selectedIconColor: PropTypes.string,
     };
 
     static defaultProps = {
       iconSize: 30,
+      iconColor: '#929292',
+      selectedIconColor: '#929292',
```

Comment:
I guess they aren't needed actually, I choose the colors based on the defaults put by react native when Icons are not rendered with 'renderAsOriginal'.